Kickstarter-iOS/Views/Cells/PledgeAmountCell.swift

```diff
@@ -36,6 +39,8 @@ final class PledgeAmountCell: UITableViewCell, ValueCell {
       |> ksr_addArrangedSubviewsToStackView()
 
     self.spacer.widthAnchor.constraint(greaterThanOrEqualToConstant: Styles.grid(3)).isActive = true
+
+    self.bindViewModel()
```

Comment:
We should consider swizzling `UITableViewCell` and do this automatically in `initWithFrame` for cells created programmatically

Reply:
Yeah, @ifbarrera and I worked on a solution for this that should work for all our programmatic views, will open a PR.